topics/ecology/tutorials/gbif_cleaning/tutorial.md

```diff
@@ -0,0 +1,367 @@
+> {:toc}
+>
+{: .agenda}
+
+# Retrive data from GBIF
+
+## Get data
+
+> <hands-on-title>Data upload</hands-on-title>
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from GBIF
+>
+> 3. **Get species occurrences data** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Scientific name of the species"*: write the scientific name of something you are interested on, for example `Loligo vulgaris`
+>    - *"Data source to get data from"*: `Global Biodiversity Information Facility : GBIF`
+>    - *"Number of records to return"*: `999999` is a minimum value
+>
+>    > <comment-title></comment-title>
+>    >
+>    > The spocc Galaxy tool allows you to search species occurences across a single or many data sources (GBIF, eBird, iNaturalist, EcoEngine, VertNet, BISON). Changing the number of records to return allows you to have all or limited numbers of occurences. Specifying more than one data source will change the manner the output dataset is formatted.
+>    >
+>    {: .comment}
+>
+>
+> 4. Check the datatype
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+> 5. Add to the output dataset (and as a good practice, each dataset you generate on Galaxy) a tag and/or a propagated tag (beginning by a `#`, here corresponding to the species (#LoligoVulgaris for example here) and/or the data source (#GBIF for example here). This will help you finding content of particular interest (using the filtering option on the history search form for example) + visualize (notably if you are using propagated tag) which dataset is associated to which content.
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+{: .hands_on}
+
+
+## Where come from the records?
+
+> <hands-on-title>Here we propose to investigate the content of the dataet looking notably at the "basisOfRecord" attribute to know more about heterogenity related to the data collection origin.</hands-on-title>
+>
+> 1. **Count** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"from dataset"*: `output` (output of **Get species occurrences data** {% icon tool %})
+>    - *"Count occurrences of values in column(s)"*: `c[17]`
+>
+>
+>    > <comment-title></comment-title>
+>    >
+>    > This tool is one of the important "classical" Galaxy tool who allows you to better synthtize information content of your data. Here we apply this tool to the 17th column (corresponding to the basisOfRecord attribute) but don't hesitate to investigate others attributes!
+>    {: .comment}
+>
+{: .hands_on}
+
+> <question-title></question-title>
+>
+> 1. How many different types of data collection origin are there?
+> 2. What is your assumption regarding this heterogeneity?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. 5
+> > 2. each basisOfRecord type is related to different collection method so different data quality
+> >
+> {: .solution}
+>
+{: .question}
+
+## Filtering data thanks to the data origin
```

Comment:
did you mean "based on" instead of "thanks to"?

Reply:
my frenglish ;)